libs/community/langchain_community/vectorstores/kinetica.py

```diff
@@ -0,0 +1,904 @@
+from __future__ import annotations
+
+import asyncio
+import enum
+import json
+import logging
+import struct
+import uuid
+from collections import OrderedDict
+from enum import Enum
+from functools import partial
+from typing import Any, Callable, Dict, Iterable, List, Optional, Tuple, Type
+
+import numpy as np
+from langchain_core.documents import Document
+from langchain_core.embeddings import Embeddings
+from langchain_core.pydantic_v1 import BaseSettings
+from langchain_core.vectorstores import VectorStore
+
+from langchain_community.vectorstores.utils import maximal_marginal_relevance
+
+
+class DistanceStrategy(str, enum.Enum):
```

Comment:
Can you reuse `langchain_community.vectorstores.utils.DistanceStrategy` instead?

Reply:
The current `pgvector` vector store implementation has this: